database/abuseemail.go

```diff
@@ -0,0 +1,180 @@
+package database
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.sia.tech/siad/build"
+)
+
+const (
+	// AbuseStatusBlocked denotes the blocked status.
+	AbuseStatusBlocked = "BLOCKED"
+
+	// AbuseStatusNotBlocked denotes the not blocked status.
+	AbuseStatusNotBlocked = "NOT BLOCKED"
+
+	// AbuseDefaultTag is the tag used when there are no tags found in the email
+	AbuseDefaultTag = "abusive"
+
+	// responseTemplateLegalNotice is a small notice we append to the response
+	// template that mentions we do not store any content on our servers
+	responseTemplateLegalNotice = `
+Please note that no content is stored on our servers, but rather on a decentralised network of hosts. 
+Therefor we are not to be held accountible for any potential abusive content they might contain.
```

Comment:
```suggestion
Therefore we are not to be held accountable for any potential abusive content they might contain.
```

I'm also not sure about `they might contain`. Who, the hosts? Might be. I would replace that with `it might contain`, "it" being the content.

Reply:
done